scripts/pipeline/orthology_benchmark.py

```diff
@@ -157,12 +167,52 @@ def get_core_names(species_names: List[str], host: str, port: int, user: str) ->
     return core_names
 
 
-def prep_input_for_orth_tools():
-    """Docstring"""
+def prep_input_for_orth_tools(source_dir: str, target_dir: str) -> None:
+    """Prepares input files for selected orthology inference tools.
 
+    Creates symlinks to input fasta files in `target_dir`.
 
-def run_orthology_tools():
-    """Docstring"""
+    Args:
+        source_dir: Path to the directory containing .fasta files.
+        target_dir: Path to the directory where symlinks to .fasta files will be created.
+
+    Raises:
+        FileExistsError: If directory `target_dir` already exists.
+        FileNotFoundError: If directory `source_dir` does not exist or does not contain any .fasta files.
+        subprocess.CalledProcessError: If creating symlinks fails for some other reason.
+    """
+    if not os.path.isdir(source_dir):
+        raise FileNotFoundError("Directory containing .fasta files not found.")
+    if len(fnmatch.filter(os.listdir(source_dir), '*.fasta')) == 0:
+        raise FileNotFoundError("No .fasta files found.")
+
+    # OrthoFinder
+    os.mkdir(target_dir)
```

Comment:
If the directory exists, it will throw `FileExistsError`. I find that more favourable than adding new symlinks/fasta files into an existing directory. In case the existing directory already contains fasta files or symlinks to fasta files, OrthoFinder would use all of them as input.